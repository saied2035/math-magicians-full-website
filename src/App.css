@import url('http://fonts.cdnfonts.com/css/rosemarine');

body {
  background: #fff;
  font-family: 'Rosemarine', sans-serif;
}
