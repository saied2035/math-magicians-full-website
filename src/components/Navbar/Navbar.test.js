import React from "react";
import renderer from 'react-test-renderer';

import App from '../../App'

it('Check all links are listed', () => {
    const tree = renderer.create(<App />).toJSON();
    expect(tree).toMatchSnapshot();
})