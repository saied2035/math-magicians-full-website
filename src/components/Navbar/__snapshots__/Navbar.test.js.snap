// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Check all links are listed 1`] = `
<div
  className="flex justify-between"
  id="Navbar"
>
  <h1
    className="ml3"
  >
    Math Magicians
  </h1>
  <nav
    className="mr5 self-center f4 b"
    id="NavbarList"
  >
    <ul
      className="list flex"
    >
      <li
        className="mr5"
      >
        <a
          aria-current="page"
          className="NavLink no-underline red active"
          href="/"
          onClick={[Function]}
        >
          Home
        </a>
      </li>
      <li
        className="mr5"
      >
        <a
          className="NavLink no-underline "
          href="/Calculator"
          onClick={[Function]}
        >
          Calculator
        </a>
      </li>
      <li
        className="mr5"
      >
        <a
          className="NavLink no-underline "
          href="/Quote"
          onClick={[Function]}
        >
          Quote
        </a>
      </li>
    </ul>
  </nav>
</div>
`;
